README.md

```diff
@@ -1,15 +1,22 @@
-# About
 <!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
 [![All Contributors](https://img.shields.io/badge/all_contributors-2-orange.svg?style=flat-square)](#contributors-)
 <!-- ALL-CONTRIBUTORS-BADGE:END -->
 
 [![CC BY 4.0][cc-by-shield]][cc-by]
 
-# Synthetic-Data
+# Welcome
 
-Content to come 
+This repository is created and maintained by the [AI for Multiple Long Term Conditions Research Support Facility](https://www.turing.ac.uk/research/research-projects/ai-multiple-long-term-conditions-research-support-facility). To find out more about this facility, take a look at the [getting started](https://github.com/aim-rsf/Getting-Started) repository. 
 
-## Contributors
+We welcome contributions from anyone, however small or large. If you choose to contribute to this synthetic data repository, please do this in line with our [code of conduct](https://github.com/aim-rsf/Getting-Started/blob/main/CODE_OF_CONDUCT.md). If you want to contribute but you're not sure where to start, see our [general guide to contributing](https://github.com/aim-rsf/Getting-Started/blob/main/CONTRIBUTING.md).
```

Comment:
I do wonder if it might be worth creating a small contributing guideline for this repository, as we can then directly link to the issues relevant to *this* repository.
The more novice the contributor, the more useful a separate contributing guideline I'd say±